x-pack/plugins/security_solution/public/common/containers/cti/event_enrichment/use_investigation_enrichment.ts

```diff
@@ -27,6 +27,7 @@ export const QUERY_ID = 'investigation_time_enrichment';
 const noop = () => {};
 const noEnrichments = { enrichments: [] };
 
+// TODO: MOVE TO FLYOUT FOLDER
```

Comment:
Please add a link to the issue that tracks those TODOs so it can be easily found when needed.

Reply:
I actually already have an entry in this overall [ticket](https://github.com/elastic/security-team/issues/7462) that takes care of that!

FYI I will take care of moving these soon (like this week or the next), once this PR is merged!

Reply:
Hi @PhilippeOberti, the point is to have references from the code to the ticket. It's frustrating to see TODOs in the code without any context. Things happen; sometimes priorities change, and sometimes TODOs can be missed in a follow-up. This can lead to obsolete, untracked TODOs in the code, which can easily be avoided by providing a link to the ticket.

Reply:
Ok that makes sense! I'll add them in my next force push. Waiting on other people to review to bundle one last commit with a rebase from latest `main` so the build is only running one more time!

Reply:
updated all TODOs in this [last commit](https://github.com/elastic/kibana/pull/187846/commits/768d6d8cbc253049be0d4df2c1dc5b7f6096c953)!